starboard/drm.h

```diff
@@ -134,15 +134,15 @@ typedef struct SbDrmSystemPrivate* SbDrmSystem;
 // |type| is the status of the session request.
 //
 // |error_message| may contain an optional error message when |status| isn't
-// |kSbDrmStatusSuccess| to provide more details about the error.  It may be
+// |kSbDrmStatusSuccess| to provide more details about the error. It may be
 // NULL if |status| is |kSbDrmStatusSuccess| or if no error message can be
 // provided.
 // |ticket| will be the same ticket that was passed to
```

Comment:
We should keep all empty lines between comments of parameters, as in this way, our docsite can list them like this (https://developers.google.com/youtube/cobalt/docs/reference/starboard/modules/drm#sbdrmsessionupdaterequestfunc), instead of put all of them as a paragraph like this (https://developers.google.com/youtube/cobalt/docs/reference/starboard/modules/media#sbmediagetmaxbuffercapacity), which is hard to read.

Fixed with adding a new line.